app/controllers/proposals_controller.rb

```diff
@@ -0,0 +1,62 @@
+class ProposalsController < ApplicationController
+  before_action :load_pq, only: [:new, :create]
+  before_action :authenticate_user, only: [:new, :create]
+
+  def new
+    @action_officers = ActionOfficer.active_list
+  end
+
+  def create
+    proposal_form = ProposalForm.new(create_params)
+    if proposal_form.valid?
+      pq = ProposalService.new.propose(proposal_form)
+      flash[:success] = 'Successfully proposed Deputy Director(s)'
+      redirect_to early_bird_landing_page_path
+    else
+      flash[:error] = 'Please choose a Deputy Director.'
```

Comment:
Would it be an option to use the ProposalForm validation messages with a re-rendered form instead of setting the flash message?